sklearn/preprocessing/tests/test_data.py

```diff
@@ -2526,3 +2526,17 @@ def test_standard_scaler_sparse_partial_fit_finite_variance(X_2):
     scaler = StandardScaler(with_mean=False)
     scaler.fit(X_1).partial_fit(X_2)
     assert np.isfinite(scaler.var_[0])
+
+
+def test_minmax_scaler_clip():
+    # test paramter 'clip' in MinMaxScaler
+    X = iris.data
+    scaler = MinMaxScaler(clip=True)
+    # default params, feature_range=(0,1)
+    X_scaled = scaler.fit(X)
+    X_min = np.min(X, axis=0)
+    X_max = np.max(X, axis=0)
+
+    X_test = [np.r_[X_min[:2], X_max[2:]]]
+    X_transformed = X_scaled.transform(X_test)
+    assert_allclose(X_transformed, [[0, 0, 1, 1]])
```

Comment:
please also check that this behaviour does not hold with clip=False.

Reply:
Hi, I have taken the PR out of the draft and have made the changes as well.